content/vue.md

```diff
@@ -15,7 +15,7 @@ Vue already has an excellent guide with many advanced topics already covered. So
 
 This documentation is purely focused on integrating it with Meteor.
 
-> Meteor has a Vue skeleton which will prepare for you a basic Vue Meteor app.
+> Meteor has a Vue skeleton which will prepare a basic Vue Meteor app for you.
```

Comment:
Preposition comes before the noun. As here it's about `you`, it is supposed to be after the object complement.